src/types.js

```diff
@@ -99,6 +95,13 @@ export type DroppableDimension = {|
     // the area in which content presses up against
     withMarginAndPadding: DimensionFragment,
   |},
+  container: {|
+    scroll: {|
+      initial: Position,
+      current: Position,
+    |},
+    page: DimensionFragment,
```

Comment:
to follow conventions in the other fragments: should this be broken into `withMargin` and `withoutMargin`?